### DISCUSSION TITLE
Requires a second button press to invoke script kit when jumping between panels in VSCode

### DISCUSSION THREAD

**Discussion by pxlprfct** [johnlindquist/kit#839] [Errors] [answered]
I've set scriptkit to `cmd + space` (Replacing Alfred 🎉🎉🎉) - but it seems to need pressing a second time to get it to open when jumping between apps, or windows within apps.

```
Hardware Overview:
  Model Name:	Mac mini
  Chip:	Apple M1
  Total Number of Cores:	8 (4 performance and 4 efficiency)
  Memory:	16 GB
  System Firmware Version:	7459.121.3
  OS Loader Version:	7459.121.3
```

https://user-images.githubusercontent.com/3811773/179402279-33291313-6e89-433d-b3ea-51e762178804.mp4

**Comment by johnlindquist** [comment]
Can you check to see if the prompt is still open behind Code when you swap focus?

The main shortcut has a feature where it will bring the UI back into focus if you're using a UI (e.g., editor, term, etc) that keeps Kit.app open when you focus on another app. I may need to check to focus logic when the shortcut is triggered.

Also, try opening a terminal and running 'tail -f ~/.kit/logs/kit.log' while you're hitting the shortcut. It might give some more insight into what's happening.

**Comment by pxlprfct** [comment, marked as answer]
Reseting the accessibility preference and quiting the app (as suggested in https://github.com/johnlindquist/kit/discussions/840) fixed it up perfectly. Works as expected now 😀
